nunaserver/requirements.txt

```diff
@@ -9,11 +9,11 @@ requests==2.25.1
 
 # Celery
 celery==5.1.0
-redis==3.5.3
+redis==4.4.4
 
 # Nunavut
-nunavut==1.3.0
-pydsdl==1.12.1
+nunavut==2.1.1
+pydsdl==1.18
```

Comment:
```suggestion
```

PyDSDL of the correct version will be pulled in through Nunavut.

Reply:
should I delete this line completely or just revert the changes?

Reply:
Please delete it